RecoTracker/PixelLowPtUtilities/src/StripSubClusterShapeTrajectoryFilter.cc

```diff
@@ -183,14 +183,14 @@ StripSubClusterShapeFilterBase::StripSubClusterShapeFilterBase(const edm::Parame
 }
 
 StripSubClusterShapeFilterBase::~StripSubClusterShapeFilterBase() {
-#if 0
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": called        " << called_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": saturated     " << saturated_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": test          " << test_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": failTooNarrow " << failTooNarrow_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": passTrim      " << passTrim_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": passSC        " << passSC_ << std::endl;
-    std::cout << "StripSubClusterShapeFilterBase " << label_ <<": failTooLarge  " << failTooLarge_ << std::endl;
+#if 1
```

Comment:
why is this activated?

Reply:
This is commented in the last commit. This was just used for testing purposes.